### ISSUE TITLE
Restructuring of Situation questions

### ISSUE THREAD

**Issue by kclimenhaga** [gryphonshafer/Quizzing-Rule-Book#168]
When I was writing questions for Acts, I was frustrated by how many of the Situation questions will be able to be jumped on formation, due to the uniqueness of the interrogatives being asked (how was it said, about whom was it said, when was it said). I don't like the idea of quizzers memorizing question lists instead of Scripture, so I tried to think of a way around it. Here's what I came up with:

Change Situation questions by removing all of the interrogatives from the question. So instead of "Question 1 is a Situation question. I need to know who said it and how", a quizmaster would just say "Question 1 is a Situation question". The quizzer would have to give all of the "situational" information in the verse in which the quote begins, or the verse prior if the quote begins at the start of a verse. The exception would be that pronouns could be clarified outside of those 1-2 verses.

**Disadvantages**
- We could not ask something like "to whom was it said" when that was stated a verse or two prior. But since some people don't think we should ask that anyhow, I don't think that's a big issue.
- We would need to establish standards around how specific they need to be (e.g. "answered/replied/said" should be considered interchangeable, in my opinion).
- It would be more difficult to write Situation questions starting several verses into a lengthier quote. Personally, I don't think we should write questions like that anyhow, but I know others disagree.
- Many of the "about whoms" would no longer work.

**Advantages**
- Quizzers would need to jump based on Scripture knowledge rather than question list knowledge.
- No more issues around "is 'stood up and said' a how?"
- No needing to choose what to ask when there are four or five interrogatives leading up to the quote.
- More clear what information is needed in the answer.

**Comment by gryphonshafer** [comment]
Setting aside the merits of this proposal for a moment, I want to highlight and address a philosophical point raised and wholeheartedly agree with it:

> "I don't like the idea of quizzers memorizing question lists instead of Scripture."

This important rules philosophy is derived from our *real* mission: **Encourage the most number of people to memorize the most number of verses.** Any rule that works counter to this mission should be anathema.

Rules should setup an environment where quizzers score better when they invest their time memorizing Scripture than through any sort of list work. This is not to say we should design rules that render list work irrelevant; rather, that rules should setup an environment where list work should always return less scoring value than memorizing Scripture. Thus, if you don't have full material memorized yet, list work is mostly a distraction; but once you have full material, list work might add differentiational value.

**Comment by jttower** [comment]
I also wholeheartedly agree with this!  Already at practice, I've been having quizzers prejumping on SIT questions because of what is being asked.  I don't like it for the same reasons you've listed.  @kclimenhaga, can you give an example question and answer in the new format you are proposing from the Acts material this year?

**Comment by kclimenhaga** [comment]
An example question would be: "Question number 1 is a Situation question. Question number 1. Question. 'Brothers and sisters, the Scripture had to be fulfilled in which the Holy Spirit spoke long ago through David concerning Judas, who served as guide for those who arrested Jesus.' " (from Acts 1:17)

The answer would be "In those days Peter stood up among the believers (a group numbering about a hundred and twenty) and said". I don't think it would have to be word perfect. I'm not sure whether we would want to require non-situational information (a group numbering about a hundred and twenty), but that's the general idea. 

(I do recognize that the answer in this case comes from the previous verse, although verse 17 starts with "and said", so my initial proposal of "just within that verse" doesn't quite work, but I'm sure we could word it somehow that would make sense.)

Currently, I have that question listed in my file as "who said it, when, and where". Why I don't like it is that a) "where" is dubious, and b) that is the only who, when, and where in my question file. "Stood up" technically can't be required information, but it definitely helps to understand the situation of what was going on when this was said.

**Comment by ARMediting** [comment]
I haven't come to a conclusion on how I feel about this, and I'm open to being persuaded, but here are my initial thoughts:

- The problem won't be a problem at Internationals since there's more material (and the rulebook is geared for Internationals). The same thing happens with smaller amounts of materials and other types of questions. Certain months will have very few MAs, FTs, or what have you. (Also, changing to this style of SIT won't fix the prejumps on SITs when quizzing on John 15-17.)
- I'd like to see the numbers run on how many question parts would get dropped because they're outside of that "verse before" range, and how many quotes quotes would become ineligible completely. Would this style provide enough questions for each narrative year?
- A less obtrusive change would be dropping some of the more obscure question types—maybe why, about whom, and how—so what's being asked for is more standard.

Also, I know we know this, any question type runs the risk of being studied from a list. I hesitate to change something so drastically because one specialty is a bit broken for one quiz meet.

**Comment by kclimenhaga** [comment]
@ARMediting Which problem do you mean "won't be a problem at Internationals"? With the Acts example I gave (who, when and where), that's the only "who, when and where" in my entire file for the whole year. So if that's the problem you're referring to, I don't think it will be fixed by Nats.

I haven't looked to see how many parts would be dropped. It depends on how the description is worded. I can look into that and get back to you.

**Comment by jswingle** [comment]
Agreed completely with Gryphon that I love the intent behind the reasoning here. I have a few thoughts:

First, when it comes to studying question lists as a preparation strategy, I think the following are all true and guide my thought process:

1. In principle, I have no problem with our competitive structure granting an incentive to study question lists.
2. However, that advantage should _only_ be accessible to quizzers who have already memorized the full material (or at least the vast majority of it - a quizzer can skip some stuff and still do very well on SIT questions and narrative year Finish/Quotes, but they're still going to be memorizing >70% of the material)
3. I think for the most part we are succeeding at #2 right now. The exception would be MAs, which do technically allow for a quizzer to memorize only some parts of the material and still specialize. I think this issue was more or less corrected when we reduced the total number of MAs to 1-2. With 2-7 many quizzers could target MAs and score well, but the idea of memorizing all the verses associated with MAs just for at max two shots per quiz is much less tantalizing. Before this rule change, there absolutely were quizzers who _just_ studied MA lists, which is a really bad thing to encourage. 
4. In principle, I also have no problem with a question being knowable before the actual text is read, but for purposes of competitive balance, this should be rare.
5. #4 presently _is_ very rare, at least at full-material competitions. There are occasional chapters with only one good MACR, and a very small number of SITs each narrative year (more on this below)
6. Regardless of whether we like any of the above points, it is impossible to totally prevent list study from being a valuable strategy. Knowing the earliest point at which things become unique is an intrinsic feature of how our rules are designed.

To follow up on point 5: with full material, there are very few SIT questions which a quizzer actually can know with certainty. In the example you gave, a WHO/WHEN/WHERE might be unique in the first few chapters of a book, but it's not a particularly uncommon thing to hear with full material. I unfortunately can't provide any exact statistics at this point, but I did specialize in SITs in 2011 John and there were maybe like _two or three_ uniquely knowable ones in the whole material. Typically, these tend to be ABOUT WHOM questions, or uncommon combinations of WHEN/WHERE/WHY. 

I have two major concerns about this proposal:

1. Restricting content to just a verse or two surrounding the quotation start is a big limitation on what we can test with SITs. The broader context is much of the point of SITs, not just the "who said it"s. If a few verses earlier the narrator establishes that the scene is in Jerusalem on the Passover Feast, then this is important information that we should want to require the quizzer to say. (By the way, I don't think there should be a 5-verse context rule with SITs. If Jesus starts talking in Jerusalem in ch. 3 and keeps going in ch. 5 with a new quotation start, I think we should require the quizzers to still know who said it and where. This is a separate proposal from this issue, obviously, but I bring this up to point out that I think we would want to err in the _other_ direction with SIT context if anything.)
2. BUT if we were to take out this one part of your proposal and still implement the idea with 5-verse context instead, SITs would become way too difficult for all but the top-tier quizzers, for obvious reasons. Only quizzers who could quote more or less the whole 5-verse surrounding context would consistently succeed.

So because of the above, how about a counter-proposal? I haven't thought it out fully yet, but it's an idea that intrigues me: **keep the QM prompts of WHO/WHERE/WHY, etc., but have the QM say them at the _end_ of the quotation instead of before beginning the quotation.**

**Comment by jswingle** [comment]
Forgot to point out what I see as the advantage to the proposal: it makes SIT jumping at higher levels of competition based on knowledge of quotation starts above the questions asked, while preserving difficulty by requiring quizzers to more or less be able to give all the contextual information if they want to consistently be counted correct. Solid material knowledge required.

But less competitive environments would still be able to answer SITs just as well. It would actually make SITs a bit easier on some quizzers, since they'd often get to hear the entire quotation read, instead of mid-tier quizzers pre-jumping. Only the best quizzers could pre-jump on them.

That being said, the QM would suddenly have to read a bunch of words, which increases likelihood of a fumble/misread and a need to throw the question out.

**Comment by jttower** [comment]
I like the idea of asking the prompts at the end of question.

**Comment by jswingle** [comment]
@jttower It would only require them to quickly quote a huge section of context if they pre-jump on it. They're still getting the same information when the whole question is read. Like all question types, it's harder with a pre-jump.

At Internationals levels of competition, where there will ALWAYS be a pre-jump, I think quizzers should aim to be prepared to quote a whole 5-verse segment within their 30 seconds. Rewarding that is a good thing.
